button.css

```
@@ -0,0 +1,19 @@
+.button{
+    width: 200px;
+    height: 45px;
+    padding: 13px 36px 15px 36px;
```

і скорочений варіант значення падінгів - 
```suggestion
    padding: 13px 36px 15px;
```